src/index.html

```diff
@@ -17,6 +17,87 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Logo"
```

Comment:
The `alt` attribute should describe the image content, not just say 'Logo'. Try to be more descriptive, like 'Company Logo'.